core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClient.java

```diff
@@ -68,4 +76,37 @@ public static BlockWorkerClient create(@Nullable Subject subject, SocketAddress
    * @throws StatusRuntimeException if any error occurs
    */
   Iterator<ReadResponse> readBlock(final ReadRequest request) throws StatusRuntimeException;
+
+  /**
+   * Creates a local block on the worker.
+   *
+   * @param responseObserver the stream observer for the server response
+   * @return the stream observer for the client request
+   */
+  StreamObserver<CreateLocalBlockRequest> createLocalBlock(
```

Comment:
The create local block RPC call has two steps:

1. the client request server to create the block. The server add the block in metadata and return the file name.
2. the client finished writing to the file. It can either send a complete request or a cancel request. the client will wait for the server to ack on the completion of the request.

Given both client and server need to send multiple messages in a single session, the API needs to be a two way streaming API.

I will add some javadoc to clarify the workflow.